functions/models/src/functions/checkInvitationCode.ts

```diff
@@ -14,3 +14,5 @@ export const checkInvitationCodeInputSchema = z.object({
 export type CheckInvitationCodeInput = z.input<
   typeof checkInvitationCodeInputSchema
 >
+
+export type CheckInvitationCodeOutput = undefined
```

Comment:
`Promise<CheckInvitationCodeOutput>` is used instead of `Promise<void>`. This change aims to guarantee consistency for consuming `engagehf-models` package. Every function now has both `Input` and `Output` type. If `Output` to something else than `undefined`, there is contract already established.